compat-data-schema.md

```diff
@@ -0,0 +1,374 @@
+# The browser-compat-data JSON format
+
+Maintained by the [MDN team at Mozilla](https://wiki.mozilla.org/MDN).
+
+## Browser compatibility information
+MDN needs the compatibility information for each feature of the Web platform. In
+order to make it easier to maintain and easy to reuse by third-party tool, we
+decided to store this data into a set of JSON files, stored in a git repository.
+
+In order to ensure the coherence of the data, we define a JSON schema that
+describes the content of the JSON files storing the info. Before accepting a
+PR, we validate the changes against this schema with _travis-ci_.
+
+This document describes the format in lay-man terms so that a human can create
+a JSON browser-compat-data file without having to decrypt the schema from
+scratch.
+
+__Note:__ The schema tries to capture as many constraints and rules about the
+browser-compat-data JSON format. Nevertheless there are constraints that cannot
+be described inside a schema, and won't be checked by automated tests. We tried
+to mark such cases in the following text with a __Not enforced by the
+schema__ notice.
+
+## The JSON format
+
+### Division in files
+The browser-compat-data schema has been designed so the division in files doesn't
+convey any meaning. That means that browser compatibility information about features
+can be stored in one single large files or being divided in smaller files.
+
+The division in separate files, themselves in different directories is guided
+by making it easy to maintain by humans.
+
+### Schema versioning
+There one single information that is tied to each file, and is mandatory. It is
+the version of schema used inside the file.
+
+    {
+      "version": "1.0.0",
+      …
+    }
+
+We are using semantic versioning, so the version is a `string` containing three
```

Comment:
Should link to http://semver.org/ - actually, IMO, we should link instead of including the three bullets below. Either we are semver, in which case the definitions must be the same, and by duplicating it we just allow for incompatibility to creep in. Or we are not, in which case we shouldn't say we are.